packages/base-map/src/index.story.js

```diff
@@ -10,6 +10,7 @@ import AllVehiclesOverlay from "../__mocks__/AllVehicles";
 import ContextMenuDemo from "../__mocks__/ContextMenuDemo";
 
 import "../../../node_modules/leaflet/dist/leaflet.css";
+import "../__mocks__/mapbox-wordmark.css";
```

Comment:
Can this be imported inside `index.js` instead? That seems a better place because the wordmark assumes a `mapbox-wordmark` defined in that CSS, and it would avoid implementers to import that CSS file.

Reply:
See PR description:

> Since the CSS for the Mapbox wordmark is somewhat large (12kb), it is included in the example

If we added it in the example index.js file, it would always be imported and add 12kb (unzipped) of unnecessary data downloads even if the implementing application didn't use a Mapbox base layer (for example, TriMet doesn't).

Reply:
Fair point.